app/controllers/jobseekers/base_controller.rb

```diff
@@ -11,4 +11,9 @@ def storable_location?
   def store_jobseeker_location!
     store_location_for(:jobseeker, request.fullpath)
   end
+
+  def update_in_progress_steps!(step)
```

Comment:
What's the reason we need to do this?

Reply:
Otherwise, the review page will claim that these details steps are 'not started'.